packages/mc-scripts/src/utils/get-session-data.spec.js

```diff
@@ -20,19 +20,25 @@ describe('When login details are correct', () => {
   beforeEach(() => {
     mockServer.use(
       rest.post(`${mcApiUrl}/tokens`, (req, res, ctx) => {
-        return res(ctx.status(200), ctx.cookie('mcAccessToken', 'hello-world'));
+        return res(
+          ctx.status(200),
+          ctx.set(
+            'set-cookie',
+            'mcAccessToken=hello-world; Expires=Tue, 15 Mar 2022 10:01:09 GMT'
```

Comment:
I would suggest to use `cookie.serialize` to have a realistic cookie value. Something like

```suggestion
            cookie.serialize('mcAccessToken', 'hello-world', { expires: new Date('2022-01-01'), httpOnly: true, domain: 'mc.europe-west1.gcp.commercetools.com' })
```